### ISSUE TITLE
Exception reporting is not showing valuable info

### ISSUE THREAD

**Issue by jgebal** [utPLSQL/utPLSQL#110]
Example:
```
create or replace package tst is
  -- %suite(Demo of documentation reporter)
  -- %test(repoting exception)
  procedure failing_exception_raised;
end;
/
create or replace package body tst is
  procedure failing_exception_raised is
    l_date date;
  begin
    l_date := to_date('abcd');
  end;
end;
/

begin
  ut_suite_manager.run_cur_schema_suites_static(ut_documentation_reporter(), a_force_parse_again => true);
end;
/
```

Will show something like:
```
Failures:
   1) repoting exception
      Failures/Errors:
        error: ORA-01858: a non-numeric character was found where a numeric was expected ORA-06512: at "UT3.UT_EXECUTABLE", line 29
               ORA-06512: at "UT3.UT_EXECUTABLE", line 70
               ORA-06512: at "UT3.UT_TEST", line 67
```

This does not show any info about exception stack trace for the called test procedure. It shows actually the stack trace of Framework (that should not be shown probably).

**Comment by jgebal** [comment]
This issue is resolved with #109 
The reporting after change is as follows:
```
Failures:
   1) repoting exception
      Failures/Errors:
        error: ORA-01858: a non-numeric character was found where a numeric was expected
               ORA-06512: at "UT3.DEMO_DOC_REPORTER1", line 21
               ORA-06512: at line 5
```
The package `UT3.DEMO_DOC_REPORTER1` is the Unit Test, so it's absolutely valid information.

**Comment by Pazus** [comment]
Closing the issue as it was resolved.
